Sources/Subs.php

```diff
@@ -4407,32 +4407,6 @@ function text2words($text, $max_chars = 20, $encrypt = false)
 	}
 }
 
-/**
- * Creates an image/text button
- *
- * @param string $name The name of the button (should be a main_icons class or the name of an image)
- * @param string $alt The alt text
- * @param string $label The $txt string to use as the label
- * @param string $custom Custom text/html to add to the img tag (only when using an actual image)
- * @param boolean $force_use Whether to force use of this when template_create_button is available
- * @return string The HTML to display the button
- */
-function create_button($name, $alt, $label = '', $custom = '', $force_use = false)
```

Comment:
Rather than deleting this function, it would be better to simply mark it as deprecated in its documentation block. That will better maintain backwards compatibility for mods and themes from 2.0.